### ISSUE TITLE
Add all in one executable to future releases

### ISSUE THREAD

**Issue by Hope-IT-Works** [nadermx/backgroundremover#71]
Hello and thank you for working on this awesome project.

It would be nice, if you could build an executable that does not need python to be installed on the system. I know that [youtube-dl](https://github.com/ytdl-org/youtube-dl) achieved this somehow.

Thank you.

**Comment by nadermx** [comment]
If some one who has done it before has a tutorial on how to attempt this, I will try

**Comment by Hope-IT-Works** [comment]
You could try it with [auto-py-to-exe](https://github.com/brentvollebregt/auto-py-to-exe).

**Comment by JoeGarcia2** [comment]
Which file would be the entry point to build the exe from?

**Comment by nadermx** [comment]
I haven't had time to try and do this, so if you want to please feel free.  Would appreciate it.  From what I understand maybe the same as the pip? backgroundremover.cmd.cli

**Comment by JoeGarcia2** [comment]
No worries. If anyone else comes across this and can point me in the right direction, this is where I'm stuck: https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe#how-do-i-convert-all-my-python-files

**Comment by HugoCortell** [comment]
I agree that an executable, probably with a UI or something, would be nice.
